src/canister/platform_orchestrator/src/api/canister_management/remove_subnet_orchestrator_from_available_list.rs

```diff
@@ -0,0 +1,17 @@
+use candid::Principal;
+use ic_cdk_macros::update;
+use shared_utils::common::utils::permissions::is_caller_controller_or_global_admin;
+
+use crate::CANISTER_DATA;
+
+
+#[update(guard = "is_caller_controller_or_global_admin")]
+fn remove_subnet_orchestrators_from_available_list(subnet_orchestrator: Principal) -> Result<String, String> {
+    CANISTER_DATA.with_borrow_mut(|canister_data| {
+        let remove_result = canister_data.subet_orchestrator_with_capacity_left.remove(&subnet_orchestrator);
```

Comment:
shouldn't we also remove from all_subnet_orchestrator_canisters_list ?

Reply:
Would be needed for upgrades.